newIDE/app/src/Utils/GetUserUID.js

```diff
@@ -0,0 +1,8 @@
+var os = optionalRequire('os');
+function getUsername(): string {
```

Comment:
It's a good idea to do this 👍

This being said, have you tried your changes?

You need to add `// @flow` at the top of the file, then to import `optionalRequire` and finally to `export` the function.
When it's done, you can import your function in the two other files.
Test that it works and it will be good.

Also rename the file `GetUsername` or better, `LocalUserInfo.js`.

Reply:
Well arthuro told me to do otherwise !

Reply:
I completly revamped it !

Reply:
done !